### PR TITLE
Restored support for "?=json" links & enabled DnD support for legacy Json in local.html

### LINKED ISSUE #1372: older &json= links don't work now?
Hi, @segun-codes @7malikk - i noticed maybe the older ?json= URLs no longer work --

https://publiclab.github.io/Leaflet.DistortableImage/examples/archive?json=https://archive.org/download/mapknitter-wayback/ceres--2.json

https://publiclab.github.io/Leaflet.DistortableImage/examples/archive?json=https://ia801508.us.archive.org/17/items/sanborn-1885/mapknitter-1676394104542.json

Was this intentional, or maybe we just missed it due to not having tests? No big deal, but I think there's still a strong case for preserving `?json=`, in case people store their JSON files somewhere else. 

No worries, this is the kind of thing tests would help us catch!

### PR DESCRIPTION
**Pull request by segun-codes**
Fixes #1372

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [X] PR is descriptively titled 📑 and links the original issue above 🔗
* [X] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [X] code is in uniquely-named feature branch and has no merge conflicts 📁
* [X] screenshots/GIFs are attached 📎 in case of UI updates
* [X] @mention the original creator of the issue in a comment below for help or for a review

**Illustration 1: DnD support for legacy json in local.html**
Illustration-1

**Illustration 2: Support "?json=" links restored**
Illustration-2

### CODE REVIEW

**Comment by segun-codes** [comment]
Hi @jywarren, this is now ready for your review (and perhaps merging).

File: examples/js/archive.js [hunk 1]
```diff
@@ -406,28 +419,38 @@ function isWorkingUrlFormat(url) {
...
+  }
 }
 
 document.addEventListener('DOMContentLoaded', async (event) => {
   if (mapReconstructionMode) {
     const url = location.href;
 
-    // working url formats: https://localhost:8080/examples/archive.html?k=texas-barnraising-copy (for mapknitter-lite generated json files)
-    // OR https://localhost:8080/examples/archive.html?kl=--10 (legacy format - for json files from legacy mapknitter.org)
+    // supported working url formats:
```
Comment (jywarren) [inline examples/js/archive.js:432]: Love these comments!!!

**Review by jywarren — APPROVED** [review]

File: examples/js/archive.js [hunk 2]
```diff
@@ -690,15 +709,17 @@ function handleDrop(e) {
...
+      if (cornerBounds.length) { // checks if the image has corners
         map.fitBounds(cornerBounds);
 
         const imgObjCollection = imgObj.collection[0][0];
         options = {
           tooltipText: imgObjCollection.tooltipText,
           corners: imgObjCollection.nodes,
         };
+
+        console.log('Archive-options.corners: ', options.corners);
```
[the remark this answers is not in the document]
Comment (segun-codes) [reply examples/js/archive.js]: Yes, now removed.

**Comment by segun-codes** [comment]
Your feedback is still being awaited... many thanks!

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Great job! Sorry it took me a bit to review!

**Pull request merged**